.buildkite/common.py

```diff
@@ -258,7 +258,9 @@ def __init__(self, with_build_step=True, **kwargs):
         # Build sharing
         if with_build_step:
             build_cmds, self.shared_build = shared_build()
-            self.build_group_per_arch("🏗️ Build", build_cmds, decorate=False)
+            self.build_group_per_arch(
+                "🏗️ Build", build_cmds, decorate=False, key="build"
```

Comment:
`key_prefix` may be a better name as the key is then derived from it.

Reply:
done!